go.mod

```diff
@@ -203,6 +203,9 @@ replace github.com/ulikunitz/xz => github.com/ulikunitz/xz v0.5.11
 // https://deps.dev/advisory/OSV/GO-2021-0053?from=%2Fgo%2Fgithub.com%252Fgogo%252Fprotobuf%2Fv1.3.1
 replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 
+// Use local kvdb package until new version is tagged.
```

Comment:
Looks like this commit fab8386b2c0cd18e041993d64497905cddde4520 is missing?

Reply:
Oh I accidentally left off the commit to update btcwallet during rebase, I'll readd it.

Reply:
I've added a commit to update the `btcwallet` dependency again to eliminate the deadlock in `waddrmgr`. Thanks for noticing it!